src/Pretix/OrderHelper.php

```diff
@@ -178,18 +184,24 @@ public function getSubEventAvailabilities(SubEvent $subEvent) {
         ->getQuotas($event, ['query' => ['subevent' => $subEvent->getId()]]);
     }
     catch (\Exception $exception) {
-      throw $this->clientException($this->t('Cannot get quotas for sub-event'),
-        $exception);
+      throw $this->clientException(
+            $this->t('Cannot get quotas for sub-event'),
+            $exception
+            );
     }
 
     foreach ($quotas as $quota) {
       try {
-        $availability = $this->pretixClient->getQuotaAvailability($event,
-          $quota);
+        $availability = $this->pretixClient->getQuotaAvailability(
+              $event,
+              $quota
+          );
       }
       catch (\Exception $exception) {
-        throw $this->clientException($this->t('Cannot get quota availability'),
-          $exception);
+        throw $this->clientException(
+              $this->t('Cannot get quota availability'),
+              $exception
+                );
```

Comment:
The coding standards may not check/require this, but please be consistent in how code is indented (cf. lines 187–190).

Reply:
Fixed